explorer/graphql/app/schema.graphql

```diff
@@ -216,12 +216,16 @@ type XMsgEdge {
 PageInfo represents pagination information
 """
 type PageInfo {
-  "Start Cursor"
-  StartCursor: BigInt!
 
-  "Next Page Cursor"
+  "Next Cursor"
+  NextCursor: BigInt!
+
+  "Has Next Page"
   HasNextPage: Boolean!
 
+  "Previous Cursor"
+  PrevCursor: BigInt!
```

Comment:
❤️

Reply:
Why is the cursor of type `BigInt!`? Is it supposed to be human-readable? I thought that cursors are usually of type `graphql.ID` 🤔. The official documentation [recommends base64 encoding](https://graphql.org/learn/pagination/#:~:text=As%20a%20reminder%20that%20the%20cursors%20are%20opaque%20and%20that%20their%20format%20should%20not%20be%20relied%20upon%2C%20we%20suggest%20base64%20encoding%20them.) them and making them unique:
> As a reminder that the cursors are opaque and that their format should not be relied upon, we suggest base64 encoding them.